spec/factories/dttp/api_placement_assignment.rb

```diff
@@ -2,8 +2,27 @@
 
 FactoryBot.define do
   factory :api_placement_assignment, class: Hash do
-    dfe_placementassignmentid { SecureRandom.uuid }
-    _dfe_contactid_value { SecureRandom.uuid }
+    transient do
```

Comment:
I've just pulled this in from https://github.com/DFE-Digital/register-trainee-teachers/pull/1787 to make merging happier.